<?xml version="1.0" encoding="utf-8"?>
<launch>

    <arg name="change_state_topic" default="/HaloA/change_state"/>

    <node name="HaloA_status_changer_node" pkg="halo_radar" type="status_changer_node.py" output="screen">
        <remap from="~topic" to="HaloA/change_state"  />
        <rosparam>
            status: transmit
            range: 50
            mode: harbor
        </rosparam>
    </node>

    <node name="HaloB_status_changer_node" pkg="halo_radar" type="status_changer_node.py" output="screen">
        <remap from="~topic" to="HaloB/change_state"  />
        <rosparam>
            status: transmit
            range: 100
            mode: offshore
        </rosparam>
    </node>
      
</launch>